gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoRestSerializer.java

```diff
@@ -134,10 +142,11 @@ public void putQueryParam(Map<String, List<String>> fields, String fieldName, Ob
   /**
    * Serializes a message to a request body in a form of JSON-encoded string.
    *
-   * @param fieldName a name of a request message field this message belongs to
    * @param fieldValue a field value to serialize
+   * @param numericEnum a boolean flag that determine if enum values should be serialized to number
+   *     or not
    */
-  public String toBody(String fieldName, RequestT fieldValue) {
```

Comment:
@vam-google The fieldName here is not used so I removed it, but I do see some logic around [how it is populated](https://github.com/googleapis/gapic-generator-java/blob/889f316d7dfcfdaf826cb8024166cc8eabeaef0d/src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceStubClassComposer.java#L844) in gapic-generator-java, can you please confirm it's safe to remove this field?

Reply:
Can you please clarify what you mean by it being removed (It is still here, from what I can see)? It not being used now should be viewed as implementation detail. On the interface it is still there and removing it would break gapic clients (i think). It also increases readability of the generated code, as it makes it clear which field actually goes to body.

Reply:
Or, by "removed" do you mean you did not put it into the overloaded method? I guess you should choose what is the proper way here. If I was implementing it, I would keep the field name even in the overloaded method (for the reasons listed above, plus it would make the interface of two methods look more similar).

Reply:
Initially I removed it from the original method, but now I added an overloaded method. It would make it more similar, but I would say it's an issue we should fix now that we have the chance to create an overloaded methods. Unless you feel strongly and think we might be able to use it in the future(even if it happens, we can still create overloaded methods at that point), I don't see much value keeping an unused argument here, it provides more confusing rather than clarity to me.

Reply:
Added the fieldName argument to the overloaded method.